pkg/interp/testdata/args.fqtest

```diff
@@ -91,6 +91,7 @@ id3v2                ID3v2 metadata
 ipv4_packet          Internet protocol v4 packet
 jpeg                 Joint Photographic Experts Group file
 json                 JSON
+macho                Mach-O is the native executable format of binaries in OS X
```

Comment:
`make actual` or `WRITE_ACTUAL=1 go test ./pkg/interp` should fix this

Reply:
Thanks for the help here

Reply:
Done thanks.